examples/graph3d/07_dot_cloud_colors.html

```diff
@@ -13,6 +13,9 @@
     var data = null;
     var graph = null;
 
+    function onclick(point) {
+      console.log(point);
```

Comment:
sometimes `point` is `null` is you click outside the chart. This should be handled in the example.